ProcessMaker/Repositories/CaseParticipatedRepository.php

```diff
@@ -65,14 +84,33 @@ public function update(CaseStarted $case, TokenInterface $token)
                 return;
             }
 
+            $processData = [
+                'id' => $token->processRequest->process->id,
+                'name' => $token->processRequest->process->name,
+            ];
+
+            $requestData = [
+                'id' => $token->processRequest->id,
+                'name' => $token->processRequest->name,
+                'parent_request_id' => $token->processRequest?->parentRequest?->id,
+            ];
+
+            $taskData = [
+                'id' => $token->getKey(),
+                'element_id' => $token->element_id,
+                'name' => $token->element_name,
+                'process_id' => $token->process_id,
+                'element_type' => $token->element_type,
+            ];
+
             $this->caseParticipated->updateOrFail([
```

Comment:
we need add error handling 

`try {
    $this->caseParticipated->updateOrFail([
        // ...
    ]);
} catch (\Exception $e) {
    Log::error('Failed to update case participated: ' . $e->getMessage());
    throw $e; // Optionally rethrow the exception if needed.
}`

Reply:
An error handling already exists in line :117